test/setupMeasure_test.js

```diff
@@ -109,5 +111,16 @@ describe('After calling the setupMeasure function of setup', () => {
             expect(dataLayer.length).toBe(1);
           });
     });
+    describe('Does not crash when unsupported strings are passed in', () => {
+      executeSnippetBeforeAndAfterSetup(
+          /* config= */
+          (measure) =>
+              measure('config', 'unusedProcessorName', {}, 'unusedStorageName',
+                  {}),
+          /* test=  */ (dataLayer) => {
+            expect(dataLayer.length).toBe(1);
+          });
+    });
```

Comment:
Anyway we can test what happens if someone passes in something that is not a constructor of one of the interfaces?

Reply:
Added a test for this. I'm not sure if we should let the library crash in these circumstances to fail loudly as soon as possible or just display a error message to the console.